include/ultrahdr/ultrahdrcommon.h

```diff
@@ -25,11 +25,31 @@
     #ifdef LOG_NDEBUG
         #include <cstdio>
 
-        #define ALOGD(...) fprintf(stderr, __VA_ARGS__)
-        #define ALOGE(...) fprintf(stderr, __VA_ARGS__)
-        #define ALOGI(...) fprintf(stdout, __VA_ARGS__)
-        #define ALOGV(...) fprintf(stdout, __VA_ARGS__)
-        #define ALOGW(...) fprintf(stderr, __VA_ARGS__)
+        #define ALOGD(...)                \
+          do {                            \
+            fprintf(stderr, __VA_ARGS__); \
+            fprintf(stderr, "\n");        \
+          } while (0);
```

Comment:
I don't think the semicolon is necessary here.

Reply:
Thanks for flagging this. Removed in the next commit in this pull request.